core-profile-tck/tck-dist/artifacts-pom.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) "2022" Red Hat and others
+  ~
+  ~ This program and the accompanying materials are made available under the
+  ~ Apache Software License 2.0 which is available at:
+  ~ https://www.apache.org/licenses/LICENSE-2.0.
+  ~
+  ~ SPDX-License-Identifier: Apache-2.0
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>jakarta.ee.tck</groupId>
+  <artifactId>core-profile-tck-dependencies</artifactId>
+  <packaging>pom</packaging>
+  <version>10.0.0-SNAPSHOT</version>
+  <name>Jakarta Core Profile TCK Dependencies</name>
+
+  <build>
+
+    <plugins>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <id>download-files</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <mkdir dir="${project.build.directory}/tcks"/>
+                <!-- download file -->
+                <get src="https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.0.zip"
+
```

Comment:
As a user that has installed the core-profile-tck distribution, is it okay if I manually update the installed copy of this file to update to new versions of jsonp-tck (e.g. to use [2.1.1.zip](https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.1.zip)?

Reply:
Yes, this is just what matches the current dependencies stated in the TCK. If the component specs release new service releases they can be used. Do we need to say something to that effect?

Reply:
I'll create a tracking issue for making this change so this pr can be merged.